### PR TITLE
Lifecycle completed draft

### PR DESCRIPTION
**Pull request by mghiumiusliu**
Full description of SDL development process.
Description entry point: "index.md"; all other articles are referenced in this document.

### CODE REVIEW

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: @mghiumiusliu,
will you update a [Config.json](https://github.com/smartdevicelink/sdl_markdown_spec#configjson) to show it on the Portal?

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: Is it Luxoft or Ford PM?

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: responSible

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: Need a break before a number list to be shown properly.

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: @mghiumiusliu 
responSibility

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: Owners is one the stack-holder, but stackerholder is more general term.
Please, see [Stakeholders definition in SAD](https://smartdevicelink.com/en/guides/core/software-architecture-document/introduction/#12-definitions-and-abbreviations).

Using `Product owner` will be more appropriate here.

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: Could we re-organise this chapter?
One monolith text looks hard for reading.

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: Could we reorder list in more easy-look way
* **mandatory** - Do1
* **mandatory** - Do2
* **mandatory** - Do3
* *optional* - Do4

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: Without `<br>` and lists this chapter shows as one big text.

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: Not clear about the `1.` and `2.`

File: docs/Lifecycle/index.md
Comment (mghiumiusliu) [inline docs/Lifecycle/index.md]: After agreement with stakeholder.
This was just a preparation for potential use on portal.
I do not have clear requirement to post it on portal so far.
If you think that name is ambiguous, I can provide separate index.md file with ToC.

File: docs/Lifecycle/index.md
Comment (mghiumiusliu) [inline docs/Lifecycle/index.md]: The one which will be assigned by stakeholders.
Currently FORD/Livio PM I assume.

File: docs/Lifecycle/index.md
Comment (mghiumiusliu) [inline docs/Lifecycle/index.md]: I would like to keep this term so far. We do have different stakeholders: budget owners, PM, technical specialists, probably some community experts. All of them have concerns and might influence on project decisions. Final responsibility take SDL PM (which is not the Product Owner as such), he will share responsibility of classical Product Owner with CCB.

File: docs/Lifecycle/index.md
Comment (mghiumiusliu) [inline docs/Lifecycle/index.md]: @pestOO 
Fixed

File: docs/Lifecycle/index.md
Comment (mghiumiusliu) [inline docs/Lifecycle/index.md]: @pestOO 
Fixed

Comment (mghiumiusliu) [inline docs/Lifecycle/index.md]: @pestOO 
Re-phrased

File: docs/Lifecycle/index.md
Comment (mghiumiusliu) [inline docs/Lifecycle/index.md]: @pestOO 
Re-phrased

File: docs/Lifecycle/ChangeControlBoard.md
Comment (pestOO) [inline docs/Lifecycle/ChangeControlBoard.md]: Nubmer list requires break before.

File: docs/Lifecycle/index.md
Comment (mghiumiusliu) [inline docs/Lifecycle/index.md]: @pestOO 
Re-organized.

File: docs/Lifecycle/ChangeControlBoard.md
Comment (pestOO) [inline docs/Lifecycle/ChangeControlBoard.md]: What the `techical tasks` mean?
Is it open-source analog of Jira CRQs?

File: docs/Lifecycle/ChangeControlBoard.md
Comment (pestOO) [inline docs/Lifecycle/ChangeControlBoard.md]: Acceptance of what? SDL delivery?
I'm not sure that it's a part of RM responsibilities.

File: docs/Lifecycle/ChangeControlBoard.md
Comment (pestOO) [inline docs/Lifecycle/ChangeControlBoard.md]: I suggest to extent to 'Update and review SAD changes'

File: docs/Lifecycle/ChangeControlBoard.md
Comment (pestOO) [inline docs/Lifecycle/ChangeControlBoard.md]: I suggest to add something like `Review code for following Design and Architectural solutions`
The main idea, that code shall not break or conflict with Arch/Design approaches.

File: docs/Lifecycle/DefinitionOfDone.md
Comment (pestOO) [inline docs/Lifecycle/DefinitionOfDone.md]: List is shown not properly

File: docs/Lifecycle/DefinitionOfDone.md
Comment (pestOO) [inline docs/Lifecycle/DefinitionOfDone.md]: The common term is Source codE without *s*.

File: docs/Lifecycle/DefinitionOfDone.md
Comment (pestOO) [inline docs/Lifecycle/DefinitionOfDone.md]: I think we can squash 1 and 2 points with link to [Pull request checklist](https://github.com/smartdevicelink/sdl_core/blob/develop/COMMITTERS.md#pull-request-checklist)

File: docs/Lifecycle/DefinitionOfDone.md
Comment (pestOO) [inline docs/Lifecycle/DefinitionOfDone.md]: Test are handled in [Scripts repo](https://github.com/smartdevicelink/sdl_atf_test_scripts)

File: docs/Lifecycle/DefinitionOfDone.md
Comment (pestOO) [inline docs/Lifecycle/DefinitionOfDone.md]: Right now it's a two different repos - core with SDD and guides with SAD.
And I recommend to split it in two different points.

File: docs/Lifecycle/DefinitionOfDone.md
Comment (pestOO) [inline docs/Lifecycle/DefinitionOfDone.md]: We have also MVC 2010/2013 compilers.

File: docs/Lifecycle/DefinitionOfDone.md
Comment (pestOO) [inline docs/Lifecycle/DefinitionOfDone.md]: I thin we can reorgonize this and following few chapters with information about code with links to SAD
https://smartdevicelink.com/en/guides/core/software-architecture-document/development-view/#481-implementation-technologies
https://smartdevicelink.com/en/guides/core/software-architecture-document/development-view/#483-development-environment-and-standards
I don't see any need to update this information in more than one place.

If you want we can rearrange these chapters in SAD.

File: docs/Lifecycle/SDLContribution.md
Comment (pestOO) [inline docs/Lifecycle/SDLContribution.md]: I don't see merging responsibilities in CCB document.

File: docs/Lifecycle/SDLContribution.md
Comment (pestOO) [inline docs/Lifecycle/SDLContribution.md]: Looks like copy-pasta issue with all following part of the file.

Comment (pestOO) [inline docs/Lifecycle/SDLContribution.md]: Except `GitMultiTri`

File: docs/Lifecycle/SDLRelease.md
Comment (pestOO) [inline docs/Lifecycle/SDLRelease.md]: Maybe we need to specify a Release version during CBB approval? (and mention it in CBB as a part of Owners/Group manger responsibility?)
IT will helps to avoid situation when someone merge feature of Next major feature to develop and mess up releasing next minor one.

File: docs/Lifecycle/SDLRelease.md
Comment (pestOO) [inline docs/Lifecycle/SDLRelease.md]: I think you mean `Owner`, because Stakeholder is ANY person who has a CONCERN in SDL.

File: docs/Lifecycle/SDLRelease.md
Comment (pestOO) [inline docs/Lifecycle/SDLRelease.md]: > as long as automated CI system
Do you mean "review and approve PRs after all successful CI builds" ?

File: docs/Lifecycle/SDLRelease.md
Comment (pestOO) [inline docs/Lifecycle/SDLRelease.md]: What is "automated CD system"?

Comment (pestOO) [inline docs/Lifecycle/SDLRelease.md]: Does it mean that some system will be auto-generates Releases?

File: docs/Lifecycle/index.md
Comment (pestOO) [inline docs/Lifecycle/index.md]: Stakeholders - ANY persons how has a CONCERN in SDL.
There is described Owners (maybe Feature owners).

**Review by pestOO — CHANGES_REQUESTED** [review]
I suggest to spit index file - it looks too complicated.
Maybe add a ToC with links to differnt sub-pages.
**Or** maybe adding numbering to chapters  (with/without adding ToC) can simplify reading.

File: docs/Lifecycle/index.md
Comment (mghiumiusliu) [inline docs/Lifecycle/index.md]: @pestOO 
This is not a plan, this is definition.
Precise names and contacts are part of the project management plan.
I think it should be a separate document which describes current management plan and roadmap of the SDL.

**Comment by LuxoftAKutsan** [comment]
@mghiumiusliu I like this changes. When we will be able to merge it to develop? 
Also is it will be possible to access from https://smartdevicelink.com/ ?

**Comment by Jack-Byrne** [comment]
Closing. This PR is quite old and the information contained might apply better to a Luxoft specific repository.

**Pull request closed without merging**
